tree/dataframe/test/TNonCopiableDS.hxx

```diff
@@ -11,7 +11,7 @@ public:
    TNonCopiable() = default;
 };
 
-class NonCopiableDS final : public ROOT::Experimental::TDF::TDataSource {
```

Comment:
It's "copyable", see e.g. http://en.cppreference.com/w/cpp/types/is_trivially_copyable

Could you correct both class and file name?